docs/core/tools/dotnet-run.md

```diff
@@ -68,6 +68,10 @@ Defines the build configuration. The default value is `Debug`.
 
 Builds and runs the app using the specified [framework](../../standard/frameworks.md). The framework must be specified in the project file.
 
+`--force`
+
+Set this flag to force all dependencies to be resolved even if the last restore was successful. This is equivalent to deleting *project.assets.json*.
```

Comment:
I usually remove the "Set this flag to" on the docs since this wasn't done consistently on the CLI.

Check https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-restore?tabs=netcore2x#options where I've done the same.